.github/workflows/dependencies.yml

```diff
@@ -0,0 +1,92 @@
+name: Check dependency updates
```

Comment:
let's Capitalise the name also 🙂

Reply:
Sure, done in e5a7db9daa05b0fa7397a2c95bd52c6d5e97d29c 🙂